### PR TITLE
Removed Mac Hydra support

### PR DESCRIPTION
**Pull request by SamGondelman**
Unfortunately, due to numerous issues with the Sixense DLL, we are forced to no longer support Hydras on Macs at all (this will only affect pre-Mavericks users, since hydras on post-Mavericks machines were broken anyways because of the DLL).

### CODE REVIEW

**Comment by SamGondelman** [comment]
Testing:

- On a Mac with Mavericks or previous, start Interface with hydras plugged in.  They should not work and no menu item should appear for them.  Try starting with the hydras unplugged, and then plug them in.  You should not crash and the hydras should continue to not work.
- On a Mac with post-Mavericks, start Interface with hydras plugged in.  They should not work and no menu item should appear for them.  Try starting with the hydras unplugged, and then plug them in.  You should not crash and the hydras should continue to not work.
- On Windows, hydras should work as before.

**Comment by ZappoMan** [comment]
you also want to change the cmake files so it doesn't even build it.... otherwise we have the "plugin" dll included but not doing anything.

**Comment by hifi-gustavo** [comment]
Build console output for commit: f495f5e3b56173d15977159300db9802fa46843d available at: https://gist.github.com/270955114355fc362c6bca1819741b00<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8122.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8122.dmg

File: cmake/macros/TargetSixense.cmake [hunk 1]
```diff
@@ -6,9 +6,11 @@
 #  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 #
 macro(TARGET_SIXENSE)
-    add_dependency_external_projects(sixense)
-    find_package(Sixense REQUIRED)
-    target_include_directories(${TARGET_NAME} PRIVATE ${SIXENSE_INCLUDE_DIRS})
-    target_link_libraries(${TARGET_NAME} ${SIXENSE_LIBRARIES})
-    add_definitions(-DHAVE_SIXENSE)
+	if(NOT APPLE)
```
Comment (ZappoMan) [inline cmake/macros/TargetSixense.cmake]: looks like your editor is inserting TABs instead of spaces

**Comment by hifi-gustavo** [comment]
Build console output for commit: 645ae3c21e96fe299ceac3c2499b218b9fc87213 available at: https://gist.github.com/270955114355fc362c6bca1819741b00<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8122.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8122.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: a9ed0b1c8386c530e08ef9e23845ca21580f05f7 available at: https://gist.github.com/270955114355fc362c6bca1819741b00<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8122.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8122.dmg

File: plugins/hifiSixense/src/SixenseManager.cpp [hunk 2]
```diff
@@ -137,6 +131,7 @@ void SixenseManager::setSixenseFilter(bool filter) {
 void SixenseManager::pluginUpdate(float deltaTime, const controller::InputCalibrationData& inputCalibrationData) {
     BAIL_IF_NOT_LOADED
 
+#ifdef HAVE_SIXENSE
```
Comment (jherico) [inline plugins/hifiSixense/src/SixenseManager.cpp]: The `HAVE_SIXENSE` macro isn't really needed.  We only build the plugin on platforms where we know we have the library available.

Comment (SamGondelman) [inline plugins/hifiSixense/src/SixenseManager.cpp]: I guess it's nice to have so you can disable it when compiling, if you really want?  Although it seems like I could get rid of it everywhere except in isSupported now.  Does that work @jherico @ZappoMan?

File: plugins/hifiSixense/src/SixenseSupportOSX.cpp [hunk 3]
```diff
@@ -1,155 +0,0 @@
...
-    return FORWARD(near_range, near_val, far_range, far_val);
-}
-
-int sixenseSetBaseColor(unsigned char red, unsigned char green, unsigned char blue) {
-    return FORWARD(red, green, blue);
-}
-int sixenseGetBaseColor(unsigned char* red, unsigned char* green, unsigned char* blue) {
-    return FORWARD(red, green, blue);
-}
-#endif
```
Comment (Atlante45) [inline plugins/hifiSixense/src/SixenseSupportOSX.cpp]: Man I was so proud of that hack ^^

**Pull request merged**